pom.xml

```diff
@@ -0,0 +1,313 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 https://maven.apache.org/xsd/maven-4.0.0.xsd">
+	<modelVersion>4.0.0</modelVersion>
+	<parent>
+		<groupId>uk.gov.companieshouse</groupId>
+		<artifactId>companies-house-parent</artifactId>
+		<version>1.3.1</version>
+	</parent>
+	<artifactId>account-validator-api</artifactId>
+	<version>unversioned</version>
+	<name>account-validator-api</name>
+	<description>Handles accounts XBRL Validator</description>
+	<properties>
+		<java.version>11</java.version>
+
+		<maven-build-helper-plugin.version>3.2.0</maven-build-helper-plugin.version>
+		<spring-boot-dependencies.version>2.6.4</spring-boot-dependencies.version>
+		<spring-boot-maven-plugin.version>2.6.4</spring-boot-maven-plugin.version>
+		<maven.compiler.source>${java.version}</maven.compiler.source>
+		<maven.compiler.target>${java.version}</maven.compiler.target>
+
+		<!-- tests -->
+		<maven-surefire-plugin.version>3.0.0-M5</maven-surefire-plugin.version>
+		<jib-maven-plugin.version>3.1.1</jib-maven-plugin.version>
+		<test-containers.version>1.16.3</test-containers.version>
+		<maven-failsafe-plugin.version>3.0.0-M5</maven-failsafe-plugin.version>
+		<skip.integration.tests>false</skip.integration.tests>
+		<skip.unit.tests>false</skip.unit.tests>
+		<maven-resources-plugin.version>2.5</maven-resources-plugin.version>
+		<mockito.version>4.8.0</mockito.version>
+
+		<!-- internal -->
+		<structured-logging.version>1.9.14</structured-logging.version>
+		<private-api-sdk-java.version>2.0.195</private-api-sdk-java.version>
```

Comment:
Why is this using the private api SDK? This will be inaccessable from external users.

Reply:
I will remove it

Reply:
I think you should just change it to be the regular [api sdk java](https://github.com/companieshouse/api-sdk-java)